### DISCUSSION TITLE
Support images or Plotly charts in chat Output

### DISCUSSION THREAD

**Discussion by codenprogressive** [langflow-ai/langflow#2460] [Q&A]
@dosu I would like to know if images could be supported in chat output?
I work on a use case where I need to output some plotly charts. Ideally, we plot those charts dynamically as plotly objects (like Streamlit or gradio) but for the short term it would be helpful if Langflow can support images in chat output.
Any workaround to achieve this?

**Comment by michelgirault** [comment]
did you were able to find a solution?

**Reply by codenprogressive** [reply]
No updates!

**Reply by berdai-co** [reply]
Did you figure it out? 
currently Agent can run python to plot function but it cant show the image or save it

**Comment by michelgirault** [comment]
the way i did, i use the custom component that save the image in a specific folder using the standard lib

**Reply by berdai-co** [reply]
Attached to final output of the message?
